src/app/services/notification/notification.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable } from '@angular/core'
+import { ToastrService } from 'ngx-toastr'
+import { ApiService } from '../api/api.service'
+
+@Injectable({
+  providedIn: 'root',
+})
+export class NotificationService {
+  private fromTime = new Date().getTime()
+
+  constructor(
+    private readonly api: ApiService,
+    private readonly toastr: ToastrService
+  ) {
+    this.getOperations()
+    setInterval(() => {
+      this.getOperations()
+    }, 10_000)
+  }
+
+  async getOperations() {
+    // TODO: Add response type
+    const data = await this.api.getOperationsSince(this.fromTime)
+    if (data.operations.length === 0) {
+      console.log('no new operations')
+      return
+    }
+    this.fromTime = new Date(data.operations[0].timestamp).getTime() + 1
+
+    data.operations.forEach((op: any) => {
+      if (op.entrypoint === 'create_auction') {
```

Comment:
Does this not spam if someone creates 10 auctions one after each other? Would remove this.

Reply:
As soon as everything is working, we will limit these notifications to items on your watchlist. But yes, if someone creates 10 auctions for 10 colors you have on your watchlist, you get 10 notifications. Depending how we show the notification history, we could also just show "You have new activity" instead.